src/test/kotlin/com/exactpro/epfast/template/dsl/JavaProperty.kt

```diff
@@ -20,8 +20,10 @@ import kotlin.reflect.KProperty
 
 fun <T> javaProperty(getter: () -> T, setter: (T) -> Unit) = JavaProperty(getter, setter)
```

Comment:
NOT NOW: Change `<T>` to `<T : Any>` (<T> is shortcut for `<T : Any?>` which allows `null`).